README.md

```diff
@@ -22,7 +22,7 @@ project.
 
 ## Toolchain setup
 
-The Apple CC toolchain in by this repository provides toolchains for
+The Apple CC toolchain in this repository provides toolchains for
 building for Apple platforms besides macOS. Since Bazel 7 this toolchain
```

Comment:
I find this first sentence to be a bit confusing. Does this toolchain exist to help us build for Apple platforms when the build platform is not MacOS?

Reply:
@keith @brentleyjones 👆

Reply:
That's right. The auto-configured CC toolchain from Bazel does not configure toolchains for iOS/watchOS/tvOS/visionOS anymore.